packages/mobile/src/identity/verification.ts

```diff
@@ -1085,6 +1089,20 @@ export function* tryRevealPhoneNumber(
         issuer,
         feeless: isFeelessVerification,
       })
+
+      if (logPhoneNumberTypeEnabled && body.phoneNumberType && body.credentials) {
```

Comment:
Do we need a feature flag to do this logging? Why not just do the logging directly?

It looks like it's adding some additional complexity for little gain.

Reply:
We were chatting with @alecps and @codyborn who supported the ability to turn off this feature if necessary as part of a conservative roll-out. Perhaps I could open an issue to remove the flag once it proves stable? Alternatively, we are fine to remove the feature flag if everyone is comfortable with that direction. Please let us know what works! Thanks.